EventListener/CampaignSubscriber.php

```diff
@@ -117,7 +117,9 @@ public function onTriggerCampaignPostSave(TriggerCampaignEvent $event): void
         $triggerCampaign = $event->getTriggerCampaign();
 
         if (isset($triggerCampaign->getChanges()['printNodeId'])) {
-            $this->getTriggerDialogService()->createCampaign($triggerCampaign);
+            $triggerCampaign = $this->getTriggerDialogService()->createCampaign($triggerCampaign);
+            $tcmr = $this->triggerCampaignModel->getRepository();
```

Comment:
Please use meaningful variable names or even better:

`$this->triggerCampaignModel->getRepository()->saveEntity($triggerCampaign);`